maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -64,16 +68,14 @@ public RepositoryConfiguration createRepository(ProjectBuildConfiguration projec
         String id = String.format(REPO_ID_FORMAT, safeUrlPart(pv.getProduct().getName()), pv.getVersion(),
                 safeUrlPart(projectBuildConfiguration.getProject().getName()), System.currentTimeMillis());
 
-        Properties properties = configuration.getModuleConfig(MAVEN_REPOSITORY_CONFIG_SECTION);
-        String baseUrl = properties.getProperty(BASE_URL_PROPERTY);
-
-        String url;
-//        try {
+        String url = "";
+        try {
+            MavenRepoDriverModuleConfig config = configuration.getModuleConfig(MavenRepoDriverModuleConfig.class);
+            String baseUrl = config.getBaseUrl();
             url = buildUrl(baseUrl, "api", "group", id);
-//        } catch (MalformedURLException e) {
-//            throw new RepositoryManagerException("Cannot format Maven repository URL. Base URL was: '%s'. Reason: %s", e,
-//                    baseUrl, e.getMessage());
-//        }
+        } catch (ConfigurationParseException e) {
+            e.printStackTrace();
```

Comment:
Can we do something better with this exception? Maybe we could handle or rethrow/wrap it?